pallets/bridge_transfer/src/lib.rs

```diff
@@ -179,122 +265,93 @@ pub mod pallet {
 			_rid: ResourceId,
 		) -> DispatchResult {
 			let source = T::BridgeOrigin::ensure_origin(origin)?;
-			<T as Config>::Currency::transfer(
-				&source,
-				&to,
-				amount,
-				ExistenceRequirement::AllowDeath,
-			)?;
-			Ok(())
-		}
-
-		/// This can be called by the bridge to demonstrate an arbitrary call from a proposal.
-		#[pallet::weight(195_000_000)]
-		pub fn lottery_handler(
-			origin: OriginFor<T>,
-			metadata: Vec<u8>,
-			_rid: ResourceId,
-		) -> DispatchResult {
-			T::BridgeOrigin::ensure_origin(origin)?;
-
-			let op = u8::from_be_bytes(
-				<[u8; 1]>::try_from(&metadata[..1]).map_err(|_| Error::<T>::InvalidCommand)?,
-			);
-			if op == 0 {
-				ensure!(metadata.len() == 13, Error::<T>::InvalidCommand);
-
-				Self::push_command(LotteryCommand::new_round(
-					u32::from_be_bytes(
-						<[u8; 4]>::try_from(&metadata[1..5])
-							.map_err(|_| Error::<T>::InvalidCommand)?,
-					), // roundId
-					u32::from_be_bytes(
-						<[u8; 4]>::try_from(&metadata[5..9])
-							.map_err(|_| Error::<T>::InvalidCommand)?,
-					), // totalCount
-					u32::from_be_bytes(
-						<[u8; 4]>::try_from(&metadata[9..])
-							.map_err(|_| Error::<T>::InvalidCommand)?,
-					), // winnerCount
-				));
-			} else if op == 1 {
-				ensure!(metadata.len() > 13, Error::<T>::InvalidCommand);
-
-				let address_len: usize = u32::from_be_bytes(
-					<[u8; 4]>::try_from(&metadata[9..13])
-						.map_err(|_| Error::<T>::InvalidCommand)?,
-				)
-				.saturated_into();
-				ensure!(
-					metadata.len() == (13 + address_len),
-					Error::<T>::InvalidCommand
-				);
-
-				Self::push_command(LotteryCommand::open_box(
-					u32::from_be_bytes(
-						<[u8; 4]>::try_from(&metadata[1..5])
-							.map_err(|_| Error::<T>::InvalidCommand)?,
-					), // roundId
-					u32::from_be_bytes(
-						<[u8; 4]>::try_from(&metadata[5..9])
-							.map_err(|_| Error::<T>::InvalidCommand)?,
-					), // tokenId
-					metadata[13..].to_vec(), // btcAddress
-				));
+			if _rid == T::ReserveTokenId::get() {
```

Comment:
```suggestion
			if rid == T::ReserveTokenId::get() {
```

The convention is that if an argument is not in use, we add "_" before its name. This time it's used. So we should rename it to `rid`